features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -220,6 +220,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.pulseaudio/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-regoheatpump" description="RegoHeatPump Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
openhab-transport-serial - this is optional and only needed if the "serial" things are used, while on the other hand if one uses the tcp/ip based things, it is not required. Do I need to specify it here in this case too?

Thanks for your review, I will adopt the changes in the next day or two...